### PR TITLE
fix: license and notice files for all the shaded artifacts

### PR DESCRIPTION
**Pull request by vermas2012**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-bigtable-hbase/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml [hunk 1]
```diff
@@ -147,6 +151,27 @@ limitations under the License.
...
+            <id>default-cli</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>add-third-party</goal>
+            </goals>
+            <configuration>
+              <excludedScopes>test,provided,system</excludedScopes>
+              <excludedArtifacts>jsr305|metrics-core|commons-logging|hbase-shaded-client|slf4j-api|slf4j-log4j12|log4j|htrace-core|findbugs-annotations|conscrypt-openjdk-uber|jaxb-api</excludedArtifacts>
+              <excludedGroups>org.bouncycastle|org.checkerframework|javax.annotation</excludedGroups>
+              <excludeTransitiveDependencies>true</excludeTransitiveDependencies>
```
Comment (igorbernstein2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml:169]: this should mirror the shade plugin config and simply include the bigtable-hbase-1x-shaded

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml [hunk 2]
```diff
@@ -130,10 +130,37 @@ limitations under the License.
...
+          <execution>
+            <id>default-cli</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>add-third-party</goal>
+            </goals>
+            <configuration>
+              <excludedScopes>test,provided,system</excludedScopes>
+              <excludedArtifacts>jsr305|metrics-core|commons-logging|hbase-shaded-client|slf4j-api|slf4j-log4j12|log4j|htrace-core|findbugs-annotations|conscrypt-openjdk-uber</excludedArtifacts>
+              <excludedGroups>org.bouncycastle|org.checkerframework|javax.annotation</excludedGroups>
```
Comment (igorbernstein2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml:157]: please add a comment this should follow the exclude rules of shade plugin and please order the exclusion in the same to make it easier to compare

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml [hunk 3]
```diff
@@ -130,10 +130,37 @@ limitations under the License.
...
+              <excludeTransitiveDependencies>true</excludeTransitiveDependencies>
```
Comment (igorbernstein2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml:158]: why is this necessary? does plugin apply excludes after resolving transitive deps?

Comment (vermas2012) [reply bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml:158]: I am not sure, the output does not change after removing it. I will remove it.

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml [hunk 4]
```diff
@@ -183,14 +210,14 @@ limitations under the License.
                   <exclude>org.slf4j:slf4j-log4j12</exclude>
                   <exclude>log4j:log4j</exclude>
                   <exclude>org.apache.htrace:htrace-core</exclude>
-                  <exclude>
-                    com.github.stephenc.findbugs:findbugs-annotations
-                  </exclude>
-                  <exclude>log4j:log4j</exclude>
```
Comment (igorbernstein2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml:189]: why did you remove the exclude for log4j?

**Review by igorbernstein2 — CHANGES_REQUESTED** [review]

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml (hunk 4 above)
Comment (vermas2012) [reply bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml:189]: duplicate. see lime 183/211

File: hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/pom.xml [hunk 5]
```diff
@@ -96,7 +96,7 @@ limitations under the License.
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+<!--                    <skip>true</skip>-->
```
Comment (vermas2012) [inline hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/pom.xml:99]: Fix this before merging.

File: hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/pom.xml [hunk 6]
```diff
@@ -161,6 +185,30 @@ limitations under the License.
...
+            <artifactId>jetty-util</artifactId>
+            <version>9.3.30.v20211001</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.eclipse.jetty</groupId>
+          <artifactId>jetty-webapp</artifactId>
+            <version>9.3.30.v20211001</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (igorbernstein2) [inline hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/pom.xml:211]: Can you replace all of this with
```<dependencyManagement> 
<dependencies>
 <dependency>
<groupId>org.eclipse.jetty</groupId>
<artifactId>jetty-bom</artifactId>
            <version>9.3.30.v20211001</version>
<type>pom</type>
            <scope>import</scope>
</dependency>
</dependencies>
</dependencyManagement>```
